static/app/views/replays/detail/network/details/onboarding.tsx

```diff
@@ -0,0 +1,162 @@
+import {ComponentProps} from 'react';
+import styled from '@emotion/styled';
+
+import {CodeSnippet} from 'sentry/components/codeSnippet';
+import ExternalLink from 'sentry/components/links/externalLink';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import useOrganization from 'sentry/utils/useOrganization';
+import useProjectSdkNeedsUpdate from 'sentry/utils/useProjectSdkNeedsUpdate';
+import type {NetworkSpan} from 'sentry/views/replays/types';
+
+export function UnsupportedOp({type}: {type: 'headers' | 'bodies'}) {
+  const title =
+    type === 'bodies'
+      ? t('Capture Request and Response Payloads')
+      : t('Capture Request and Response Headers');
+
+  return (
+    <StyledInstructions>
+      <h1>{title}</h1>
+      <p>
+        {tct(
+          `This feature is only compatible with [fetch] and [xhr] request types. [link]`,
+          {
+            fetch: <code>fetch</code>,
+            xhr: <code>xhr</code>,
+            link: (
+              <ExternalLink href="https://github.com/getsentry/sentry-javascript/issues/7103">
```

Comment:
Just wanted to confirm, this is a placeholder URL?

Reply:
Placeholder url until we have docs yes. There are two ExternalLinks in this PR that would need to be updated.